recipes-bsp/u-boot/u-boot-juno.bb

```diff
@@ -0,0 +1,18 @@
+require ${COREBASE}/meta/recipes-bsp/u-boot/u-boot.inc
+
+SUMMARY = "U-Boot bootloader for Arm Development Platform Juno VExpress"
+
+LICENSE = "GPLv2+"
+LIC_FILES_CHKSUM = "file://Licenses/README;md5=a2c678cfd4a4d97135585cad908541c6"
+
+SRC_URI = "git://git.linaro.org/landing-teams/working/arm/u-boot.git;protocol=https;branch=17.04"
+SRCREV = "24c6ee8c9d6f0768ced0a564191b0d6676922550"
+
+PV = "v2017.03+git${SRCPV}"
```

Comment:
branch=17.04, PV=17.03?

Reply:
yeah, Linaro 17.04 release is based on U-Boot 2017.03 release.